stingray/pulse/tests/test_modeling.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from stingray.pulse.modeling import fit_sinc, fit_gaussian
+
+np.random.seed(0)
+
```

Comment:
Huh, curious. I'm sure those tests didn't show up last time. Either I'm still asleep or the website didn't load properly. I give odds of ~50:50 :)